web/html/src/manager/minion/proxy/proxy-config.tsx

```diff
@@ -487,20 +408,13 @@ export function ProxyConfig({ serverId, parents, currentConfig, initFailMessage
                     labelClass="col-md-3"
                     divClass="col-md-6"
                   />
-                  <Select
+                  <Text
                     name="registryTftpdTag"
                     label={t("TFTPD Tag")}
+                    placeholder={t("e.g., " + registryTagExample)}
```

Comment:
It's preferable to avoid string concat when dealing with translations, the reason for this is that this leaves translators little room to work with in languages where the word order differs. A better way to achieve the same result would be

```ts
t("e.g. {registryTagExample}", {
  registryTagExample
})
```

This way the translators can work with the full string and change where the actual value is put.